dev/reference/tools/data-migration/upgrade.md

```diff
@@ -9,195 +9,131 @@ aliases: ['/docs/dev/how-to/upgrade/data-migration/']
+>
+> For this particular upgrade operation, it is not supported to just upgrade a part of the components. Otherwise, an error might occur.
 
-## Upgrade to v1.0.0-133-g2f9fe82
+## Upgrade to v1.0.0-rc.1-12-gaa39ff9
 
 ### Version information
 
 ```bash
-Release Version: v1.0.0-133-g2f9fe82
-Git Commit Hash: 2f9fe827d668add6493b2a3da107e0a01b94c6d1
+Release Version: v1.0.0-rc.1-12-gaa39ff9
+Git Commit Hash: aa39ff981dfb3e8c0fa4180127246b253604cc34
 Git Branch: dm-master
-UTC Build Time: 2018-12-19 04:58:46
+UTC Build Time: 2019-07-24 02:26:08
 Go Version: go version go1.11.2 linux/amd64
 ```
 
 ### Main changes
 
-Before this version, `mysql-instances` in the task configuration file (`task.yaml`) contains the following information:
-
-- `config`: the upstream MySQL address, user name and password.
-- `instance-id`: identifies an upstream MySQL.
-
-Starting from this version, the above two kinds of information are removed and the following information is added:
-
-- `source_id`: exists in `inventory.ini`, used to identify an upstream MySQL instance or a master-slave replication group.
-- `source-id`: exists in `mysql-instances` of the task configuration file. Its value corresponds to `source_id` of `inventory.ini`.
-
-> **Note:**
->
-> To guarantee that the checkpoint information of the existing task stored in the downstream database can still be used, the value of `source_id`or `source-id` should be consistent with that of `instance-id` of the corresponding DM-worker.
-
-### Upgrade operation example
-
-1. Modify the `inventory.ini` configuration information.
-
-    Set the corresponding `source_id` for all DM-worker instances.
-
-    For example, modify
-
-    ```ini
-    dm-worker1 ansible_host=172.16.10.72 server_id=101 mysql_host=172.16.10.72 mysql_user=root mysql_password='VjX8cEeTX+qcvZ3bPaO4h0C80pe/1aU=' mysql_port=3306
-    ```
-
-    to
-
-    ```ini
-    dm-worker1 ansible_host=172.16.10.72 source_id="mysql-replica-01" server_id=101 mysql_host=172.16.10.72 mysql_user=root mysql_password='VjX8cEeTX+qcvZ3bPaO4h0C80pe/1aU=' mysql_port=3306
-    ```
-
-2. Use DM-Ansible to perform a rolling update on DM.
-
-3. Modify the task configuration file (`task.yaml`).
-
-    Remove the `config` and `instance-id` configuration items and add the `source-id` configuration item (corresponding to `source_id` in `inventory.ini`).
-
-    For example, modify
+Starting from this release, TiDB DM checks all configurations rigorously. Unrecognized configuration triggers an error. This is to ensure that users always know exactly what the configuration is.
 
-    ```yaml
-    config:
-          host: "192.168.199.118"
-          port: 4306
-          user: "root"
-          password: "1234"
-    instance-id: "instance118-4306" # It is unique. It is used as the ID when storing the checkpoint, configuration and other information.
-    ```
+### Upgrade notes
```

Comment:
It's good that you modify this heading based on the inappropriate Chinese version. 👍